_posts/2019-02-14-Viewability.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Measuring and delivering viewability
+subtitle: Is an add delivered, if nobody can see it?
+category: data
+tags: [data science, ml, frontend]
+author: Bastian Kronenbitter
+header-img: "images/home-bg.jpg"
+---
+
+Hello. Let me start by telling you, that viewability is somewhat of a pet peeve of mine. First of all, the word is dumb. It is in artificial creation of the advertisement industry, which due to some strange reason does not want to use the perfectly fine "visibility".
+And viewability measurement and delivery should be the absolute standard nowadays. According to the [definition of the IAB](https://www.iab.com/wp-content/uploads/2015/06/MRC-Viewable-Ad-Impression-Measurement-Guideline.pdf), an advertisement is considered viewable (*crinch*) when more than 50% of its content were in the viewport for more than one second.
+From my point of view, this should be the absolute minimum for an ad to be "delivered".
```

Comment:
suggest "From my point of view, this should be the absolute minimum requirement for an ad to be considered "delivered".